holidays/countries/uruguay.py

```diff
@@ -36,60 +35,64 @@ def _populate(self, year):
         self[date(year, JAN, 1)] = "Año Nuevo [New Year's Day]"
 
         # Día de los Trabajadores.
-        self[date(year, MAY, 1)] = "Día del Trabajo [Labour Day]"
+        self[
+            date(year, MAY, 1)
+        ] = "Día de los Trabajadores [International Workers' Day]"
 
         # Jura de la Constitución.
-        self[date(year, JUL, 18)] = "Jura de la constitución"
+        self[
+            date(year, JUL, 18)
+        ] = "Jura de la Constitución [Constitution Day]"
 
         # Declaratoria de la Independencia.
         self[
             date(year, AUG, 25)
         ] = "Día de la Independencia [Independence Day]"
 
         # Christmas.
-        self[date(year, DEC, 25)] = "Navidad [Christmas]"
+        self[date(year, DEC, 25)] = "Día de la Familia [Day of the Family]"
 
         # Partially paid holidays:
 
-        # Día de Reyes - Feriado en el cual se conmemora la llegada de
-        # los reyes magos a Jesus.
-        self[date(year, JAN, 6)] = "Día de Reyes"
+        # Día de los Niños.
+        self[date(year, JAN, 6)] = "Día de los Niños [Children's Day]"
 
         # Natalicio de José Gervacio Artigas.
```

Comment:
I believe it should be `Gervasio` instead of `Gervacio` here and below.

Reply:
Yes, exactly. Thank you!